src/app/code/community/Divante/VueStorefrontIndexer/Model/Indexer/Datasource/Category/Attributes.php

```diff
@@ -159,6 +159,23 @@ private function prepareCategory(array $categoryDTO)
             'created_at',
             'updated_at',
             'request_path',
+            'description',
+            'meta_description',
+            'custom_layout_update',
+            'meta_title',
+            'filter_price_range',
+            'include_in_menu',
+            'custom_use_parent_settings',
+            'custom_apply_to_products',
+            'top_category',
+            'show_products_from_category',
+            'used_in_product_search',
+            'custom_design',
+            'page_layout',
+            'landing_page',
+            'custom_design_from',
+            'custom_design_to',
+            'meta_keywords'
```

Comment:
by default all category attributes should be exported to elastic. We are using in different project attributes like 'description', 'meta_descriptions', etc.
@pkarw what do you think ?

There should be a configuration to set up it a different way (and choose which attributes should be exported)
or you should move this to new module (+ maybe some refactoring will be needed so you can easily override this)

Reply:
I agree that we should export all category attributes to Elastic

Reply:
Well, the problem is that, when having +500 categories, the category tree gets quite large, therefore the HTML document size on VSF gets humongous (3-4 MBs), slowing down the page rendering. We discovered that by not indexing all those presumably uneeded attributes, reduces the document size considerably. I agree there should be a way to dynamically define those attributes (in backoffice) but that actually requires a bit more  coding. A more convenient way could be probably to let those attributes be XML-config defined. What do you think ?

Reply:
@qbo-tech we've addressed this problem in 1.7 0- where we're getting just the first level of categories into SSR generated HTML + then dynamically load the next level (without `children_data` properties); so what I'd like to say is You can safely index all attributes